app/main/views/services.py

```diff
@@ -144,8 +144,11 @@ def update_service(service_id):
     update_details = validate_and_return_updater_request()
     update = validate_and_return_service_request(service_id)
 
-    updated_service = update_and_validate_service(service, update)
+    # Check for an update to the copied_to_following_framework flag on the service object
+    if 'copiedToFollowingFramework' in update and isinstance(update['copiedToFollowingFramework'], bool):
```

Comment:
It would probably be nicer to throw a 400 if it's not a bool rather than silently ignore it.

Reply:
It would. I was looking at adding it to one of the schemas but the service data validation happens elsewhere, and the `validate_and_return_updater_request` and `validate_and_return_service_request` are just checking for `updated_by` and the top level `services` key. 

tl:dr I'll add a quick commit to raise on non-bools.